platform/base/core/src/main/java/com/peregrine/replication/DefaultReplicationMapperService.java

```diff
@@ -131,21 +128,22 @@ void modified(Configuration configuration) {
     private void setup(final Configuration configuration) {
         init(configuration.name(), configuration.description());
         // Register this service as Replication instance
-        logger.trace("Default Mapping: '{}'", configuration.defaultMapping());
-        Map<String, Map<String, String>> temp = splitIntoParameterMap(new String[] {configuration.defaultMapping()}, ":", "\\|", "=");
-        logger.trace("Mapped Default Mapping: '{}'", temp);
+        log.trace("Default Mapping: '{}'", configuration.defaultMapping());
+        var temp = splitIntoParameterMap(configuration.defaultMapping(), ":", "\\|", "=");
```

Comment:
Is it necessary to remove the variables type? It's more informative before. Otherwise difficult to know this is map of maps?